include/aspect/parameters.h

```diff
@@ -623,6 +623,7 @@ namespace aspect
      */
     int                            checkpoint_time_secs;
     int                            checkpoint_steps;
+    double                         walltime_from_resume;
```

Comment:
What is this parameter for? I don't see why you need it.

Reply:
I added it because I added "wall_time_resume" to "save_critical_parameters". I've removed both additions now but as I mention below I'm not sure where I'm supposed to save "walltime_from_resume" then so that this value is read in again on a restart? Thanks!